blogposts/2020/3.18/announcing-sourcegraph-3.18.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.18: Search across multiple revisions and non-master branches, custom search pages, campaigns now support GitLab, new C++ precise code intelligence"
+author: Laureen Hudson
+publishDate: 2020-MM-20T10:00-07:00
+tags: [blog]
+slug: sourcegraph-3.18
+heroImage: /blog/3.18-release-blog-img.jpg
+published: true
+---
+
+We’re improving Sourcegraph universal code search continuously, to help you solve the Big Code problems you face, all day every day. Here's how we're doing it.
+
+-   [Indexed non-master branches](https://about.sourcegraph.com/blog/indexed-non-master-branches)
+
+    Often, you need to search code that isn’t in your default branch, like long-lived release branches or important tags. In Sourcegraph 3.18, we now use an index on the default branch to speed up search operations, with the added capability to index branches other than the default branch.
```

Comment:
- We've always used an index on the default branch to speed up search. That isn't new. The new part is the "added capability" mentioned here.
- "search operations" sounds redundant/awkward, kind of like "code search engine" vs "code search". Just saying "speed up search" is fine and more common.
- The value here is "you can search non-default branches much faster now". The indexing is just the "how", and I don't think we convey the value clearly here. I think the 2nd sentence here needs to make the point clearly, something like "You can now index multiple branches (not just the default branch) to make these kinds of searches much faster."

Reply:
done